packages/@ember/-internals/glimmer/tests/integration/components/error-handling-test.js

```diff
@@ -1,166 +1,170 @@
+import { DEBUG } from '@glimmer/env';
+
 import { moduleFor, RenderingTestCase, runTask } from 'internal-test-helpers';
 
 import { set } from '@ember/-internals/metal';
 
 import { Component } from '../../utils/helpers';
 
-moduleFor(
-  'Errors thrown during render',
-  class extends RenderingTestCase {
-    ['@test it can recover resets the transaction when an error is thrown during initial render'](
```

Comment:
Since this PR is not attempting to change the production experience, this test (error occurs, and still rerenders) should exist in the else (eg still tested in prod).